src/runtime_src/core/pcie/emulation/hw_em/alveo_shim/shim.cxx

```diff
@@ -746,6 +748,20 @@ namespace xclhwemhal2 {
         }
         setenv("VITIS_KERNEL_PROFILE_FILENAME", kernelProfileFileName.c_str(), true);
         setenv("VITIS_KERNEL_TRACE_FILENAME", kernelTraceFileName.c_str(), true);
+
+        const char* ldisplay = std::getenv("DISPLAY");
+        if(!ldisplay)
+        {
+          if (mLogStream.is_open())
+          {
```

Comment:
Please be consistent.  Line 753 is missing a space, line 755 is okay.  Why the difference?

Reply:
Agreed. The space is added in if condition.